TestAssets/TestPackages/dotnet-dependency-tool-invoker/DotnetDependencyToolInvokerParser.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Cli.Utils;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Tools.DependencyInvoker
+{
+    internal static class DotnetDependencyToolInvokerParser
+    {
+        public static Microsoft.DotNet.Cli.CommandLine.Command DotnetDependencyToolInvoker() =>
+            Create.Command(
+                "dotnet-dependency-tool-invoker",
+                "DotNet Dependency Tool Invoker",
+                Accept.ExactlyOneArgument()
+                      .With(name: "COMMAND",
+                            description: "The command to execute."),
+                false,
+                Create.Option(
+                    "-h|--help",
+                    "Show help information",
+                    Accept.NoArguments(),
+                    materialize: o => o.Option.Command().HelpView()),
```

Comment:
This `materialize` argument has been removed from the latest parser version and previously wasn't used for anything in any case.